doc/content/verification_and_validation/val-2d.md

```diff
@@ -0,0 +1,175 @@
+| $A$       | * area of Tungsten sample              | 0.0025                                                      | m                     | [!cite](hino1998hydrogen) |
+| $l$       | thickness of Tungsten sample         | 1 $\times 10^{-4}$                                          | m                     | [!cite](hino1998hydrogen) |
+| $T_l$     | lowest temperature                   | 300                                                         | K                     | [!cite](hino1998hydrogen) |
+| $T_h$     | highest temperature                  | 1273                                                        | K                     | [!cite](hino1998hydrogen) |
+| $k_T$     | heating rate          | 50                                                          | K/min                 | [!cite](hino1998hydrogen) |
+
+
+!alert note title=The $\chi^2$ is adjusted to better correspond to experimental results
+TMAP7 ([!cite](ambrosek2008verification)) adjusts the maximum atom fraction as 0.0041 in trap 2 from [!cite](anderl1992deuterium) to better correspond to experimental results. Thus, we also adjust it to 0.00175 to correspond to experimental results from [!cite](hino1998hydrogen).
+
+## Results
+
+In this case, there is a general background drift on desorption flux due to an increasing source of atoms going into the gas phase as the heated region spread with time. Thus, we add a ramped signal peaking at 4.87 $\times 10^{17}$ H$_2$/m$^2$/s to the results of the TMAP8 during the thermal desorption.
+[val-2d_comparison] shows the comparison of the TMAP8 calculation and the experimental data. There is reasonable agreement between the TMAP predictions and the experimental data with the root mean square percentage error of RMSPE = 32.83 %.
+Note that the agreement could be improved by adjusting the model parameters and adding more potential traps. TMAP7 is limited to three traps, but TMAP8 can introduce an arbitrarily number of trapping populations. It is also possible to perform this optimization with [MOOSE's stochastic tools module](https://mooseframework.inl.gov/modules/stochastic_tools/index.html).
+
+There are several reasons for the no exact fit with the data from [!cite](hino1998hydrogen): the most prominent one is the two-dimensionality of the experiment arising from beam non-uniformity and radial diffusion [!citep](anderl1992deuterium). The actual trap energies are probably a little lower than the ones indicated above if the time lag caused by two-dimensionality is significant. Exchange of hydrogen with chamber surfaces, particularly the sample support structure, may also be a factor.
+
+One reason the measured signal falls off after \approx 6300 s while the computed one remains steady is that the source of additional atoms in the experiment may be an expanding area that grow non-linearly, while the sample is being heated but stopped growing and thus stops emitting when the heating stops.
+
+!media comparison_val-2d.py
```

Comment:
The TMAP8 results do not look that great right now. 
It looks like the time steps are really large and the curve does not look smooth.